### PR TITLE
[MIG] website_project migrated into new api

### PR DESCRIPTION
**Pull request by atp-odoo**
Task: https://www.odoo.com/web#id=12128&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.X87BCsZgOuBPOdud

### CODE REVIEW

File: addons/website_project/__init__.py [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
 import models
-import controllers
\ No newline at end of file
+import controllers
```
Comment (atp-odoo) [inline addons/website_project/__init__.py]: order import

File: addons/website_project/__openerp__.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-OpenERP Projects
+Odoo Projects
 ================
 
         """,
-    'author': 'OpenERP SA',
+    'author': 'Odoo SA',
     'depends': ['website_mail', 'project'],
     'data': [
         'views/website_project.xml',
```
Comment (atp-odoo) [inline addons/website_project/__openerp__.py]: remove qweb tag. no static/src/xml file

File: addons/website_project/controllers/main.py [hunk 3]
```diff
@@ -1,32 +1,12 @@
...
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+from openerp import http
+from openerp.http import request
 
-from openerp.addons.web import http
-from openerp.addons.web.http import request
 
 class website_project(http.Controller):
```
Comment (atp-odoo) [inline addons/website_project/controllers/main.py]: CamelCase class name

File: addons/website_project/models/__init__.py [hunk 4]
```diff
@@ -1 +1 @@
-import website_project
\ No newline at end of file
+import website_project
```
Comment (atp-odoo) [inline addons/website_project/models/__init__.py]: Add # -*- coding: utf-8 -*-

File: addons/website_project/models/website_project.py [hunk 5]
```diff
@@ -1,9 +1,12 @@
-from openerp.osv import orm
+# -*- coding: utf-8 -*-
+from openerp import models
 
-class project(orm.Model):
+
+class project(models.Model):
     _name = 'project.project'
```
Comment (atp-odoo) [inline addons/website_project/models/website_project.py]: Camelcase for class name

File: addons/website_project/models/website_project.py [hunk 6]
```diff
@@ -1,9 +1,12 @@
...
-    _inherit = ['project.project','website.seo.metadata']
+    _inherit = ['project.project', 'website.seo.metadata']
+
 
-class task(orm.Model):
+class task(models.Model):
```
Comment (atp-odoo) [inline addons/website_project/models/website_project.py]: same as above use camel case for class name

File: addons/website_project/__openerp__.py (hunk 2 above)
Comment (hna-odoo) [inline addons/website_project/__openerp__.py]: done.

File: addons/website_project/controllers/main.py (hunk 3 above)
Comment (hna-odoo) [inline addons/website_project/controllers/main.py]: done.

File: addons/website_project/models/__init__.py (hunk 4 above)
Comment (hna-odoo) [inline addons/website_project/models/__init__.py]: done.

File: addons/website_project/models/website_project.py (hunk 6 above)
Comment (hna-odoo) [inline addons/website_project/models/website_project.py]: done.

File: addons/website_project/models/website_project.py (hunk 5 above)
Comment (hna-odoo) [inline addons/website_project/models/website_project.py]: done.

**Comment by rgo-odoo** [comment]
- Rename,
    - website_project.py ==> project.py
    - website_project.xml ==> project_templates.xml

**Pull request closed without merging**